### PR TITLE
fix: change link color with mode

### PR DESCRIPTION
**Pull request by ojafero**
# Description

Link color will now change with theme mode.

## Type of change

[comment]: # 'Please delete options that are not relevant.'

- [x] Bug fix (non-breaking change which fixes an issue)

# Checklist:

- [x] I have performed a self-review of my own code

### CODE REVIEW

File: src/pages/500.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import { useTheme } from '@emotion/react';
```
Comment (RubenSmn) [inline src/pages/500.js:1]:
```js
import { useTheme } from '@mui/material/styles';
```

**Review by RubenSmn — CHANGES_REQUESTED** [review]

**Review by RubenSmn — CHANGES_REQUESTED** [review]
I think that an even better approach is to only use the `Link` component with a `sx` prop where we get the theme from. That way we don't need to use inline-styles and don't have a `a` tag in a `a` tag

**Comment by ojafero** [comment]
@RubenSmn That arrangement is the recommended approach https://nextjs.org/docs/upgrading#link-component

**Comment by RubenSmn** [comment]
Ah I see, but we're using the `Link` from `@mui/material` @ojafero

**Review by RubenSmn — APPROVED** [review]

**Pull request merged**